src/test/resources/soop_sst_trajectory.xml

```diff
@@ -6,68 +6,849 @@
+        <attribute name="flag_values" value="'B, C, D, E, F, G, H, J, K, L, M, Q, S, T, U, V, X, Z'"/>
+        <attribute name="flag_meaning" value="'Value_out_of_bounds Time_not_sequential Failed_T_Tw_Td_test Failed_true_wind_recomputation_test Platform_velocity_unrealistic Value_exceeds_threshold Discontinuity Erroneous_value Suspect_value_(visual) Value_located_over_land Instrument_malfunction Pre-flagged_as_suspect Spike_in_data_(visual) Time_duplicate Suspect_value_(statistical) Step_in_data_(statistical) Spike_in_data_(statistical) Value_passed_all_tests'"/>
       </attributes>
     </variable>
     <variable>
       <name>LONGITUDE</name>
+      <encoder>double</encoder>
+      <dimensions>
+        <dimension name="TIME"/>
+      </dimensions>
+      <attributes>
+        <attribute name="standard_name" value="'longitude'"/>
+        <attribute name="long_name" value="'longitude'"/>
+        <attribute name="units" value="'degrees east'"/>
+        <attribute name="axis" value="'X'"/>
+        <attribute name="reference_datum" value="'geographical coordinates, WGS84 projection'"/>
+        <attribute name="_FillValue" value="999999."/>
+        <attribute name="valid_min" value="-180."/>
+        <attribute name="valid_max" value="180."/>
+        <attribute name="ancillary_variables" value="'LONGITUDE_quality_control'"/>
+      </attributes>
+    </variable>
+    <variable>
+      <name>LONGITUDE_quality_control</name>
+      <encoder>char</encoder>
+      <dimensions>
+        <dimension name="TIME"/>
+      </dimensions>
+       <attributes>
+        <attribute name="standard_name" value="'longitude status_flag'"/>
+        <attribute name="long_name" value="'quality flag for longitude'"/>
+        <attribute name="quality_control_set" value="3."/>
+        <attribute name="quality_control_conventions" value="'BOM (SST and Air-Sea flux) quality control procedure'"/>
+        <attribute name="flag_values" value="'B, C, D, E, F, G, H, J, K, L, M, Q, S, T, U, V, X, Z'"/>
+        <attribute name="flag_meaning" value="'Value_out_of_bounds Time_not_sequential Failed_T_Tw_Td_test Failed_true_wind_recomputation_test Platform_velocity_unrealistic Value_exceeds_threshold Discontinuity Erroneous_value Suspect_value_(visual) Value_located_over_land Instrument_malfunction Pre-flagged_as_suspect Spike_in_data_(visual) Time_duplicate Suspect_value_(statistical) Step_in_data_(statistical) Spike_in_data_(statistical) Value_passed_all_tests'"/>
+       </attributes>
+    </variable>
+        <variable>
+      <name>ATMP</name>
+      <encoder>float</encoder>
+      <dimensions>
+        <dimension name="TIME"/>
+      </dimensions>
+      <attributes>
+       	<attribute name="standard_name" value="'air_pressure'"/>
+		<attribute name="long_name" value="'pressure (measured variable) exerted by the atmosphere'"/>
+		<attribute name="units" value="'millibar'"/>
+		<attribute name="_FillValue" value="-9999.f"/>
```

Comment:
All measured variables should include a 'coordinates' variable attribute :
<attribute name="coordinates" value="'TIME LATITUDE LONGITUDE'"/>